examples/hitl/rearrange_v2/rearrange_v2.py

```diff
@@ -44,6 +44,26 @@
 from habitat_hitl.environment.hablab_utils import get_agent_art_obj_transform
 from habitat_sim.utils.common import quat_from_magnum, quat_to_coeffs
 
+try:
+    from habitat_llm.agent.env import (
+        register_actions,
+        register_measures,
+        register_sensors,
+    )
+except ImportError:
+    print("Habitat-LLM not installed. Skipping import.")
+
+
+# -------- HABITAT-LLM SPECIFIC CODE --------
+
+
+def register_habitat_llm_extensions(config):
+    register_actions(config)
+    register_measures(config)
+    register_sensors(config)
+
+
+# -------- HABITAT-LLM SPECIFIC CODE --------
```

Comment:
We could probably move this to [this file](https://github.com/facebookresearch/habitat-lab/pull/1933), and change it from an "episode loader" to a "`habitat-llm` loader".